django_project/certification/models/attendee.py

```diff
@@ -0,0 +1,66 @@
+# coding=utf-8
+"""
+Attendee model definitions for certification apps
+"""
+
+from django.core.urlresolvers import reverse
+from django.db import models
+from core.settings.contrib import STOP_WORDS
+from unidecode import unidecode
+from certificate import Certificate
+from django.utils.text import slugify
+
+
+class Attendee(models.Model):
+    """Course Attendee model."""
+
+    firstname = models.CharField(
+        help_text="First name course attendee.",
+        max_length=200,
+        null=False,
+        blank=False
+    )
+
+    surname = models.CharField(
+        help_text="Surname course attendee.",
+        max_length=200,
+        null=False,
+        blank=False
+    )
+
+    email = models.CharField(
+        help_text="Email address.",
+        max_length=200,
+        null=False,
+        blank=False
+    )
+
+    slug = models.SlugField()
+    objects = models.Manager()
+    certificate = models.ForeignKey(Certificate)
```

Comment:
i think it is many to many, because attendee can have zero or many of certificates.